documentation/source/testbench_tools.rst

```diff
@@ -5,26 +5,26 @@ Test Bench Tools
 Logging
 =======
 
-Cocotb extends the python logging library. Each dut, monitor, driver, and scoreboard (as well as any other function using the **coroutine** decorator) implements it's own logging object, and each can be set to it's own logging level.
+Cocotb extends the python logging library. Each dut, monitor, driver, and scoreboard (as well as any other function using the **coroutine** decorator) implements it's own logging object, and each can be set to it's own logging level. Within a dut, each heirarchical object can also have individual logging levels set.
```

Comment:
Typo: heirarchical

Reply:
I would use ``coroutine``, not **coroutine** (but that style is not used consistently in the cocotb docs, so could use a general round of cleanups).

Reply:
I'll switch. Hopefully my search and replacing didn't go overboard in hitting the other locations.